vpn-proxy/app/models.py

```diff
@@ -135,3 +153,82 @@ def save(self, *args, **kwargs):
     def delete(self, *args, **kwargs):
         self.stop()
         super(Tunnel, self).delete(*args, **kwargs)
+
+
+class Ports(models.Model):
```

Comment:
Perhaps this should be renamed to PortForwarding or just Forwarding, port isn't very descriptive and the model contains much more info than just a port number.